bin/index.js

```diff
@@ -9,13 +9,13 @@ const fs = promisifyAll(require('fs'));
 const program = require('commander')
     .version(pkg.version)
     .description(`${pkg.name} (v${pkg.version}): ${pkg.description}`)
-    .usage('<file>')
-    .arguments('<file>')
+    .usage('<file> [options]')
+    .option('-o, --output', 'write to HTML file')
     .parse(process.argv);
 
 const srcFilename = path.join(program.args[0]);
 
 fs.readFileAsync(srcFilename, 'utf8')
```

Comment:
this does not need to be one long Promise chain. It may be easier to split it up:

```javascript
const format = fs.readFileAsync(srcFilename, 'utf8')
  .then(readme => formatter(readme));

if (program.output) {
  format.then(/* ... */)
} else {
  format.then(/* ... */)
}
```